site/content/en/docs/user/metrics-configuration.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Metrics"
+---
+
+# Metrics Configuration
+
+{{< hint "info" >}}
+
+This document walks you through how to configure the Metrics feature.
+
+{{< /hint >}}
+
+## What is a Metrics?
+
+The [Metrics] is a [`kwok` Configuration][configuration] that allows users to define and simulate metrics endpoints exposed by kubelet.
+
+The YAML below shows all the fields of a Metrics resource:
+
+``` yaml
+kind: Metrics
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: <string>
+spec:
+  path: <string>
+  metrics:
+  - name: <string>
+    help: <string>
+    kind: <string>
+    dimension: <string>
+    labels:
+    - name: <string>
+      value: <string>
+    value: <string>   # for counter and gauge
+    buckets:          # for histogram
+    - le: <float64>
+      value: <string>
+      hidden: <bool>
+```
+
+There are total four metric-related endpoints in kubelet: `/metrics`, `/metrics/resource`, `/metrics/probe` and `/metrics/cadvisor`,
+all of which are exposed with a Prometheus style. The Metrics resource is capable of simulating endpoints with such style.
+
+To simulate a metric endpoint, first, you need to specify the RESTful `path` of the endpoint,
+which will be installed and exposed by the metric service of `kwok` at port `10247` after applied.
+The `path` must start with `/metrics`, otherwise, `kwok` will not install it.
+
+
+{{< hint "info" >}}
+Starting from metrics-server 0.7.0, it is allowed to specify the path to scrape metrics for a node.
+Specifically, metric-server will check if a node has annotation `metrics.k8s.io/resource-metrics-path` 
+and use it as the target metric scrape path. Combing with the Metric CR, the feature makes it possible to integrate
+`kwok` and metrics-server easily. For a fake node, by adding that annotation and setting its value to the `path`
+specified in a Metric resource, metrics-server will collect data from the endpoints exposed by `kwok` instead of
+scrapping from kubelet.
```

Comment:
```suggestion
Starting from metrics-server 0.7.0, it is allowed to specify the path to scrape metrics for a node.
Specifically, metrics-server will check if a node has annotation `metrics.k8s.io/resource-metrics-path` and use it as the target metric scrape path.
Combined with the Metric CR, the feature makes it possible to integrate `kwok` and metrics-server.
For a fake node, by adding that annotation and setting its value to the `path`
specified in a Metric resource, metrics-server will collect data from the endpoints exposed by `kwok` instead of
scrapping from kubelet.
```